File: comu/com.energyict.comserver.comserver.core.extjs/src/main/web/js/mdc/src/audit/model/AuditReference.js
/*
 * Copyright (c) 2019 by Honeywell International Inc. All Rights Reserved
 */

Ext.define('Mdc.audit.model.AuditReference', {
    extend: 'Ext.data.Model',

    fields: [
        {name: 'name', type: 'auto'},
        {name: 'reference', type: 'auto'}
    ]
});

File: copl/com.elster.jupiter.audit/src/main/java/com/elster/jupiter/audit/AuditReference.java
/*
 * Copyright (c) 2019 by Honeywell International Inc. All Rights Reserved
 */

package com.elster.jupiter.audit;

import aQute.bnd.annotation.ProviderType;

@ProviderType
public interface AuditReference {

    String getName();

    Object getContextReference();

    boolean isRemoved();
}

